docs/ruleset_functions/get_zone_BPF_BAT.md

```diff
@@ -0,0 +1,142 @@
+ 
+**Function Call:** 
+- None
+
+## Logic:  
+
+- create a dictionary to map lighting space types to BPF building area types: ```building_area_lookup = { 
+	ATRIUM_LOW_MEDIUM: UNDETERMINED,							
+	ATRIUM_HIGH: UNDETERMINED,
+	AUDIENCE_SEATING_AREA_AUDITORIUM: UNDETERMINED,
+	AUDIENCE_SEATING_AREA_CONVENTION_CENTER: UNDETERMINED,
+	AUDIENCE_SEATING_AREA_EXERCISE_CENTER: UNDETERMINED,
+	AUDIENCE_SEATING_AREA_GYMNASIUM: UNDETERMINED,
+	AUDIENCE_SEATING_AREA_MOTION_PICTURE_THEATER: UNDETERMINED,
+	AUDIENCE_SEATING_AREA_PENITENTIARY: ALL_OTHER,
+	AUDIENCE_SEATING_AREA_PERFORMING_ARTS_THEATER: UNDETERMINED,
+	AUDIENCE_SEATING_AREA_RELIGIOUS_FACILITY: UNDETERMINED,
+	AUDIENCE_SEATING_AREA_SPORTS_ARENA: UNDETERMINED,
+	AUDIENCE_SEATING_AREA_TRANSPORTATION_FACILITY: UNDETERMINED,
+	AUDIENCE_SEATING_AREA_ALL_OTHER: UNDETERMINED,
+	BANKING_ACTIVITY_AREA: UNDETERMINED,
+	CLASSROOM_LECTURE_HALL_TRAINING_ROOM_PENITENTIARY: ALL_OTHER,
+	CLASSROOM_LECTURE_HALL_TRAINING_ROOM_SCHOOL: SCHOOL,
+	CLASSROOM_LECTURE_HALL_TRAINING_ROOM_ALL_OTHER:	UNDETERMINED,
+	CONFERENCE_MEETING_MULTIPURPOSE_ROOM: UNDETERMINED,
+	CONFINEMENT_CELLS: ALL_OTHER,
+	COPY_PRINT_ROOM: UNDETERMINED,
+	CORRIDOR_FACILITY_FOR_THE_VISUALLY_IMPAIRED: UNDETERMINED,
+	CORRIDOR_HOSPITAL: HEALTHCARE_HOSPITAL,
+	CORRIDOR_MANUFACTURING_FACILITY: ALL_OTHER,
+	CORRIDOR_ALL_OTHERS: UNDETERMINED,
+	COURT_ROOM:	ALL_OTHER,
+	COMPUTER_ROOM: UNDETERMINED,
+	DINING_AREA_PENITENTIARY: ALL_OTHER,
+	DINING_AREA_FACILITY_FOR_THE_VISUALLY_IMPAIRED: UNDETERMINED,
+	DINING_AREA_BAR_LOUNGE_OR_LEISURE_DINING: RESTAURANT,
+	DINING_AREA_CAFETERIA_OR_FAST_FOOD_DINING: RESTAURANT,
+	DINING_AREA_FAMILY_DINING: RESTAURANT,
+	DINING_AREA_ALL_OTHERS: RESTAURANT,
+	ELECTRICAL_MECHANICAL_ROOM: UNDETERMINED,
+	EMERGENCY_VEHICLE_GARAGE: UNDETERMINED,
+	FOOD_PREPARATION_AREA: UNDETERMINED,
+	GUEST_ROOM: HOTEL_MOTEL,
+	JUDGES_CHAMBERS: ALL_OTHER,
+	DWELLING_UNIT: MULTIFAMILY,
+	LABORATORY_EXCEPT_IN_OR_AS_A_CLASSROOM: ALL_OTHER,
+	LAUNDRY_WASHING_AREA: UNDETERMINED,
+	LOADING_DOCK_INTERIOR: UNDETERMINED,
+	LOBBY_FACILITY_FOR_THE_VISUALLY_IMPAIRED: UNDETERMINED,
+	LOBBY_ELEVATOR: UNDETERMINED,
+	LOBBY_HOTEL: HOTEL_MOTEL,
+	LOBBY_MOTION_PICTURE_THEATER: ALL_OTHER,
+	LOBBY_PERFORMING_ARTS_THEATER: ALL_OTHER,
+	LOBBY_ALL_OTHERS: UNDETERMINED,
+	LOCKER_ROOM: UNDETERMINED,
+	LOUNGE_BREAKROOM_HEALTH_CARE_FACILITY: HEALTHCARE_HOSPITAL,
+	LOUNGE_BREAKROOM_ALL_OTHERS: UNDETERMINED,
+	OFFICE_ENCLOSED: OFFICE,
+	OFFICE_OPEN_PLAN: OFFICE,
+	PARKING_AREA_INTERIOR: UNDETERMINED,
+	PHARMACY_AREA: HEALTHCARE_HOSPITAL,
+	RESTROOM_FACILITY_FOR_THE_VISUALLY_IMPAIRED: UNDETERMINED,
+	RESTROOM_ALL_OTHERS: UNDETERMINED,
+	SALES_AREA: RETAIL,
+	SEATING_AREA_GENERAL: UNDETERMINED,
+	STAIRWELL: UNDETERMINED,
+	STORAGE_ROOM_HOSPITAL: HEALTHCARE_HOSPITAL,
+	STORAGE_ROOM_SMALL: UNDETERMINED,
+	STORAGE_ROOM_LARGE: UNDETERMINED,
+	VEHICULAR_MAINTENANCE_AREA: ALL_OTHER,
+	WORKSHOP: UNDETERMINED,
+	ASSISTED_LIVING_FACILITY_CHAPEL: MULTIFAMILY,
```

Comment:
I am not sure that assisted living is classed as multifamily, suggest UNDETERMINED